device_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+const fileMode = os.FileMode(0640)
```

Comment:
Maybe you could call this `fileMode0640` because `fileMode` could be anything and it does not help when reading the code.